inventory/helpers.go

```diff
@@ -122,22 +122,50 @@ func addMySqldExporter(t *testing.T, body agents.AddMySqldExporterBody) *agents.
 	t.Helper()
 	res, err := client.Default.Agents.AddMySqldExporter(&agents.AddMySqldExporterParams{
 		Body:    body,
-		Context: context.TODO(),
+		Context: pmmapitests.Context,
 	})
 	assert.NoError(t, err)
 	require.NotNil(t, res)
 	return res.Payload
 }
 
-func assertEqualAPIError(t *testing.T, err error, expectedCode int) bool {
+func addMongoDBExporter(t *testing.T, body agents.AddMongoDBExporterBody) *agents.AddMongoDBExporterOKBody {
 	t.Helper()
-	expectedError := &runtime.APIError{
-		OperationName: "unknown error",
-		Code:          expectedCode,
+	res, err := client.Default.Agents.AddMongoDBExporter(&agents.AddMongoDBExporterParams{
+		Body:    body,
+		Context: pmmapitests.Context,
+	})
+	assert.NoError(t, err)
+	require.NotNil(t, res)
+	return res.Payload
+}
+
+func assertEqualAPIError(t *testing.T, err error, expectedCode int64, expectedError string) bool {
```

Comment:
Let's try to use larger interface that `error` there for more compile-time checks.
Also can we use `server.ErrorResponse` there for expected error?